playbooks/adhoc/uninstall.yml

```diff
@@ -21,6 +21,7 @@
 
     - set_fact:
         is_atomic: "{{ ostree_output.rc == 0 }}"
+        is_containerized: is_atomic or containerized | default(False)
```

Comment:
Moved this into openshift.common.is_atomic and openshift.common.is_containerized